### PR TITLE
Updating APIM version to rc1

### PR DESCRIPTION
**Pull request by pubudu538**
### Issues

Fixes https://github.com/wso2/product-microgateway/issues/3276, https://github.com/wso2/product-microgateway/issues/3248, https://github.com/wso2/product-microgateway/issues/3293, https://github.com/wso2/product-microgateway/issues/3295

### Automation tests
 - Unit tests added: Yes/No
 - Integration tests added: Yes/No

### Tested environments

Not Tested

---
#### Maintainers: Check before merge
- [x] Assigned 'Type' label
- [x] Assigned the project
- [x] Validated respective github issues
- [x] Assigned milestone to the github issue(s)

### CODE REVIEW

File: integration/test-integration/pom.xml [hunk 1]
```diff
@@ -315,7 +315,7 @@
                 </os>
             </activation>
             <properties>
-                <apictl.package>apictl-${apictl.version}-macosx-x64.tar.gz</apictl.package> <!-- Use same amd64 distribution until arm64 available -->
+                <apictl.package>apictl-${apictl.version}-darwin-arm64.tar.gz</apictl.package> <!-- Use same amd64 distribution until arm64 available -->
```
Comment (renuka-fernando) [inline integration/test-integration/pom.xml:318]: Let's remove the comment as well.

```suggestion
                <apictl.package>apictl-${apictl.version}-darwin-arm64.tar.gz</apictl.package>
```

File: resources/k8s-artifacts/choreo-connect/choreo-connect-deployment.yaml [hunk 2]
```diff
@@ -188,9 +188,6 @@ spec:
         - name: enforcer-keystore-vol
           secret:
             secretName: enforcer-keystore
-        - name: config-toml-vol
```
Comment (renuka-fernando) [inline resources/k8s-artifacts/choreo-connect/choreo-connect-deployment.yaml:191]: Let's remove `logconfig-toml-vol` as well

**Review by renuka-fernando — DISMISSED** [review]

File: integration/test-integration/pom.xml (hunk 1 above)
Comment (pubudu538) [reply integration/test-integration/pom.xml]: Fixed with https://github.com/wso2/product-microgateway/pull/3294/commits/5ce86bde4b42be78841f82e5ad112d7eed5c9c35

File: resources/k8s-artifacts/choreo-connect/choreo-connect-deployment.yaml (hunk 2 above)
Comment (pubudu538) [reply resources/k8s-artifacts/choreo-connect/choreo-connect-deployment.yaml:191]: Fixed with https://github.com/wso2/product-microgateway/pull/3294/commits/5ce86bde4b42be78841f82e5ad112d7eed5c9c35

**Review by renuka-fernando — APPROVED** [review]

**Pull request merged**